draft-ietf-quic-transport.md

```diff
@@ -1351,6 +1355,14 @@ connection when they move networks.  This includes state that can be hard to
 recover such as outstanding requests, which might otherwise be lost with no easy
 way to retry them.
 
+An endpoint that receives packets that contain a source IP address and port that
```

Comment:
Also, reordering can easily cause one to receive a packet from the new path, then then old path, then the new path.  I think it'd be helpful to provide some advice about sending in response to the packet with the largest packet number?